adapters/audienceNetwork/facebook.go

```diff
@@ -39,6 +41,10 @@ type facebookReqExt struct {
 	AuthID     string `json:"authentication_id"`
 }
 
+const (
+	consentedProviders = "consented_providers"
```

Comment:
IMHO there's not much value in this constant declaration. Directly using the string in code would be fine, depending on how you feel about "magic strings".

Reply:
Modified to "magic string"